packages/ERTP/src/paymentLedger.js

```diff
@@ -197,11 +229,16 @@ export const makePaymentLedger = (
       const [payment] = moveAssets(
         harden([srcPayment]),
         harden([srcPaymentBalance]),
+        optRecoverSet,
       );
       return payment;
     });
   };
 
+  /** @type {IssuerClaim} */
+  const claim = (paymentP, optAmountShape = undefined) =>
```

Comment:
Keep this code somewhere, but all the claim operations in our code are in tests. Because of Zoe, all real usage ends up depositing into Zoe escrow. So all the `claim` and related operations can be removed, leaving just the recovery logic around the `withdraw` operation.

Reply:
Git keeps the code. I really doubt we will need it again. It is sufficiently easy to reproduce anyway that I'm just "deleting" it.